includes/html/graphs/device/sla_jitter-loss-percent.inc.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * LibreNMS module to Graph Cisco IPSLA UDP Jitter metrics
+ *
+ * Copyright (c) 2016 Aaron Daniels <aaron@daniels.id.au>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$sla = dbFetchRow('SELECT `sla_nr` FROM `slas` WHERE `sla_id` = ?', [$vars['id']]);
```

Comment:
```php
$sla_nr = Sla::where('id', $vars['id'])->value('sla_nr');

if ($sla_nr) {
$sla['sla_nr'] -> $sla_nr
```

Reply:
Ah I wondered about the dBFetchRow check failing. Afraid I'm not sure how to implement this. Assume some further logic required instead of `...`?
Will `use App\Models\Sla` also be required?

Reply:
Yes, the use would be required.  You should be able to just paste my code in.

Reply:
The ... is your existing code. and change any references of $sla['sla_nr'] to  $sla_nr